src/Omega_h_adios2.cpp

```diff
@@ -12,6 +12,20 @@ namespace Omega_h {
 
 namespace adios {
 
+Omega_h_Type getOmegahDataType(const std::string &dataType)
+{
+  if (dataType == "int8_t")
+    return OMEGA_H_I8;
+  if (dataType == "int32_t")
+    return OMEGA_H_I32;
+  if (dataType == "int64_t")
+    return OMEGA_H_I64;
+  if (dataType == "double")
+    return OMEGA_H_F64;
+
+  throw std::invalid_argument("Equivalent Omegah data type is not available for " + dataType);
```

Comment:
Please use fail(...) here https://github.com/SCOREC/omega_h/blob/09a51716b37c630ab04be85e50ff2f8014383f5d/src/Omega_h_fail.hpp#L35 as it will throw only when exceptions are enabled and abort otherwise.